www/web/src/components/ThemeManager.astro

```diff
@@ -0,0 +1,65 @@
+---
+type Props = {
+	defaultTheme?: "auto" | "dark" | "light" | undefined;
+};
+
+const { defaultTheme = "auto" } = Astro.props;
+---
+
+<script is:inline data-default-theme={defaultTheme}>
+  window.theme ??= (() => {
+    const defaultTheme =
+      document.currentScript.getAttribute("data-default-theme");
+    const storageKey = "theme";
+    const store =
+      typeof localStorage !== "undefined"
+        ? localStorage
+        : { getItem: () => null, setItem: () => {} };
+
+    const mediaMatcher = window.matchMedia("(prefers-color-scheme: light)");
+    let systemTheme = mediaMatcher.matches ? "light" : "dark";
+    mediaMatcher.addEventListener("change", (event) => {
+      systemTheme = event.matches ? "light" : "dark";
+      applyTheme(theme.getTheme());
+    });
+
+    function applyTheme(theme) {
+      const resolvedTheme = theme === "auto" ? systemTheme : theme;
+      document.documentElement.dataset.theme = resolvedTheme;
+      if (resolvedTheme === "light") {
+        document.documentElement.classList.remove("dark");
```

Comment:
Don’t need to add the class here

Reply:
Hey Oliver, I did not understand what you meant by that. can you share a suggestion for this part for my clarity?

Reply:
In step two on this guide on the apply theme function you'll see we don't add class list, maybe I should ask why does this add class list as I'm not sure if it's actually being used 

https://astro-tips.dev/recipes/dark-mode/#theme-manager-component

Reply:
Just applied your suggestion, I went with that classList method just cause I was following another theme. Resolved with latest commit [`9b55d2c` (#81)](https://github.com/astrolicious/studiocms/pull/81/commits/9b55d2c3a5acf26f6c3b461939ef2d8c7908c3a8)